### ISSUE TITLE
Setting property to null when deep == true causes error.

### ISSUE THREAD

**Issue by triploc** [elbywan/hyperactiv#3]
Awesome library.  Awesome updates to support array

let obj = observe({ }, { deep: true });
obj.a = 10;
obj.a = null;

**Comment by elbywan** [comment]
> Awesome library. Awesome updates to support array

@triploc Thanks !!

```
let obj = observe({ }, { deep: true });
obj.a = 10;
obj.a = null;
````

Sorry about this, rookie mistake.

Just published version `0.1.6`, which contains the fix 👍

**Comment by triploc** [comment]
Thanks dude!
